docker-compose.yaml

```diff
@@ -107,7 +107,7 @@ services:
       - AWS_DEFAULT_REGION
       - AWS_ENDPOINT_URL
       - S3_BUCKET
-      - PYTHONPATH=/mzai/lumigator/python/mzai/backend
+      - PYTHONPATH=/mzai/lumigator/python/mzai/backend:/mzai/lumigator/python/mzai/jobs
```

Comment:
Instead of changing the PYTHONPATH directly, maybe we can use the features from uv: https://docs.astral.sh/uv/concepts/projects/dependencies/#path

Reply:
Thanks for the link! I actually had looked at that option, but we don't currently meet any of those preqrequisites because the job is not a package (no wheel, no zip, no pyproject.toml). Since each job may have its own dependencies, I think I would need to add a pyproject.toml to each job folder and then add a link to each individual job in the backend. Seemed like more overhead and one more place that woud need to be touched every time you add a new job definition, so I decided against it.

However, one thing I could do is to move this PYTHONPATH env var out of the docker-compose and instead put it into the Dockerfile. That might be a bettter place for the PYTHONPATH to be set anyways.